modules/postgresql/src/main/java/org/testcontainers/containers/PostgreSQLContainer.java

```diff
@@ -46,6 +46,8 @@ public PostgreSQLContainer(final String dockerImageName) {
     @Override
     protected void configure() {
         addExposedPort(POSTGRESQL_PORT);
+        // Disable Postgres driver use of java.util.logging to reduce noise at startup time
+        withUrlParam("loggerLevel", "OFF");
```

Comment:
I haven't investigated what exactly this does, but it looks a bit weird. I can understand if by default the logging level should not be DEBUG or so, but I would definitely like to see errors and probably also warnings by default (i.e. without having to investigate how to change this manually).

Please disregard if I missed something.

Reply:
It used to be in the `getJdbcUrl` method like this:
```java
return "jdbc:postgresql://" + getContainerIpAddress() + ":" + getMappedPort(POSTGRESQL_PORT) + "/" + databaseName + "?loggerLevel=OFF";
```
I just moved it to the `configure()` method. 

I'm guessing the logger was turned off on purpose to remove the noise at the start of the container.